runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java

```diff
@@ -85,20 +89,37 @@ public void onWidgetHidden(WidgetHiddenChanged event)
 		}
 
 		String text = widget.getText();
+		final IntStream spriteIDs =
+			Arrays.stream(widget.getParent().getDynamicChildren()).mapToInt(Widget::getSpriteId);
```

Comment:
You most likely do not need int stream at all, can't you like just `map` it instead?

Reply:
Hmm, maybe, idk to me it feels like unnecessary transformation. But if it does not have downsides, then its ok I guess.